src/Languages/TypeScript.ts

```diff
@@ -369,5 +369,6 @@ export class TypeScript extends CLikeLanguage {
         variables.null = "undefined";
         variables.typesAfterName = true;
         variables.typeLeft = ": ";
+        variables.typecheck = "typeof";
```

Comment:
The names you used in the design doc are good. Instead of hacking together equivalents of them from existing properties, I'd prefer if you made a new set of properties under `variables`, `NullProperties`, that had those properties.

It's a little more overhead in the short-term but will give GLS more flexibility to adapt to new languages in the future. It's stayed with very similar OOP languages so far but may branch out to something that doesn't have the same format of null checking.